lib/test_suite.py

```diff
@@ -125,10 +125,10 @@ def __init__(self, suite_path, args):
             self.ini[i] = dict.fromkeys(self.ini[i].split()) \
                 if i in self.ini else dict()
         for i in ["lua_libs"]:
-            self.ini[i] = map(
-                lambda x: os.path.join(suite_path, x),
-                dict.fromkeys(self.ini[i].split())
-                if i in self.ini else dict())
+            lua_libs = self.ini.get(i, "")
+            self.ini[i] = []
+            for p in lua_libs.split():
+                self.ini[i].append(os.path.join(suite_path, p))
```

Comment:
Nitpicking (sorry). There is a kind of inaccuracy: the loop body is supposed to be option unaware, but it uses certain option name (`lua_libs`) inside. Since we're here I would rewrite it with proper names and leave a comment. So Une may just add an option of the same format here.

```python
# For given options in the following format:
#   
#  | key = v1 v2
#   
# Set self.ini[key] to the following value:
# ['<suite_name>/v1', '<suite_name>/v2'].
for option_name in ["lua_libs"]:
    option_raw_value = self.ini.get(option_name, "") 
    self.ini[option_name] = []
    for item in option_raw_value.split():
        self.ini[option_name].append(os.path.join(suite_path, item))
```

However you can leave it unchanged due to usage of 'fork' start method. Even if you'll apply it, it'll be just refactoring and should not be part of this patchset. So it is just note, ignore it.

Reply:
Filed #272 for this.